components/Layout/AudioControls.vue

```diff
@@ -58,77 +129,106 @@
             <v-btn disabled icon class="mx-1">
               <v-icon>mdi-repeat-off</v-icon>
             </v-btn>
-          </div>
-          <time-slider />
+          </v-col>
+        </v-row>
+        <div
+          v-if="isFullScreenPlayer && nextTrackName"
+          class="d-flex justify-center align-center"
+        >
+          <h4 class="text-h6 font-weight-thin font-italic">
+            {{ $t('upNext') }}: {{ nextTrackName }}
+          </h4>
         </div>
-      </v-col>
-      <v-col cols="3" class="d-none d-md-flex align-center justify-end">
-        <v-tooltip top>
-          <template #activator="{ on, attrs }">
-            <v-btn disabled icon class="mr-2" v-bind="attrs" v-on="on">
-              <v-icon>mdi-playlist-play</v-icon>
-            </v-btn>
-          </template>
-          <span>{{ $t('queue') }}</span>
-        </v-tooltip>
-        <volume-slider />
-        <v-tooltip top>
-          <template #activator="{ on, attrs }">
-            <v-btn disabled icon class="ml-2" v-bind="attrs" v-on="on">
-              <v-icon>mdi-fullscreen</v-icon>
-            </v-btn>
-          </template>
-          <span>{{ $t('fullScreen') }}</span>
-        </v-tooltip>
-      </v-col>
-      <v-col cols="3" class="d-flex d-md-none px-0 align-center justify-end">
-        <v-btn icon>
-          <v-icon>mdi-heart</v-icon>
-        </v-btn>
-        <v-btn icon @click="togglePause">
-          <v-icon>
-            {{ isPaused ? 'mdi-play' : 'mdi-pause' }}
-          </v-icon>
-        </v-btn>
-      </v-col>
-    </v-row>
-  </v-container>
+      </v-container>
+    </v-footer>
+  </transition>
 </template>
 
 <script lang="ts">
-import { ImageType } from '@jellyfin/client-axios';
+import { BaseItemDto, ImageType, RepeatMode } from '@jellyfin/client-axios';
 import Vue from 'vue';
 import { mapActions, mapGetters } from 'vuex';
-import imageHelper from '~/mixins/imageHelper';
 import timeUtils from '~/mixins/timeUtils';
+import imageHelper from '~/mixins/imageHelper';
 import { PlaybackStatus } from '~/store/playbackManager';
 
 export default Vue.extend({
   mixins: [timeUtils, imageHelper],
   computed: {
-    ...mapGetters('playbackManager', ['getCurrentItem']),
-    runtime(): number {
-      return this.ticksToMs(this.getCurrentItem.RunTimeTicks) / 1000;
+    footerColor(): string | undefined {
+      if (this.isFullScreenPlayer) {
+        return 'rgba(0,0,0,0.15)';
+      } else {
+        return undefined;
+      }
     },
     isPaused(): boolean {
       return this.$store.state.playbackManager.status === PlaybackStatus.paused;
+    },
+    isPlaying(): boolean {
+      return (
+        this.$store.state.playbackManager.status !== PlaybackStatus.stopped
```

Comment:
Should isPlaying be true if there is an error? Wouldn't it be better to check if  `this.$store.state.playbackManager.status` is equal to  `PlaybackStatus.playing`?

Reply:
@Camc314 That doesn't cover pause.